test/context2d.test.js

```diff
@@ -380,12 +380,12 @@ describe("Context2D", ()=>{
       expect(pixel(10, 10)).toEqual([0, 162, 213, 245])
     })
 
-    test('shadow', async() => {      
```

Comment:
I'm getting more trailing whitespaces in my diffs... is your editor supposed to trim those? If we're not trimming, LMK so I can disable that in my editor for this project. I typically don't leave those differences in my commits (for clarity), but it's getting to be a chore to filter them out. Thanks!

Reply:
Sorry about that (I know it's annoying). I noticed my editor wasn't trimming whitespace midway through #172 but wanted to wait until it's complete to have a single 'format & remove whitespace' commit (vs turning on whitespace trimming now and having a bunch of random changes just in the files i've recently saved).

Reply:
Ah, well no worries, luckily we can stage partial changes so just a minor annoyance. Bummer about the editor settings... I hate it when that happens!